nuget/Microsoft.Windows.CppWinRT.props

```diff
@@ -43,6 +43,7 @@ Copyright (C) Microsoft Corporation. All rights reserved.
             <InterfaceIdentifierFileName Condition="'%(Midl.InterfaceIdentifierFileName)'==''">nul</InterfaceIdentifierFileName>
             <ProxyFileName Condition="'%(Midl.ProxyFileName)'==''">nul</ProxyFileName>
             <TypeLibraryName Condition="'%(Midl.TypeLibraryName)'==''"></TypeLibraryName>
+            <NoMidl>true</NoMidl>
```

Comment:
This name feels a bit too generic.  You are still calling `midl.exe` but are adding/removing the `/nomidl` option to it.  This _might_ be misinterpreted as skipping the call to `midl.exe` entirely.  Consider using a more specific name such as `<PassNoMidlOption>` (that one isn't great but hopefully you get the idea)

Reply:
Yeah, I'm still confused at what `/nomidl` does to begin with.

Reply:
I don't disagree but the /nomidl option name itself already makes little sense to me. Naming it this way at least enables easy matching to command line options.

Reply:
Actually according to the docs /nomidl does exactly what it says. It doesn't run midl.exe, see https://learn.microsoft.com/en-us/windows/win32/midl/-nomidl

Reply:
Keep in mind that `MIDL.EXE` and `MIDLRT.EXE` are two very different things. The `/nomidl` makes sense if you think about it as a `MIDLRT` option that determines whether it will delegate to `MIDL` for further processing. The internals are a little muddied, since `MIDL` can also delegate to `MIDLRT`, but that's the idea at least. If all you need is a .winmd then `/nomidl` is what you want since that's something that `MIDLRT` can handle without `MIDL` getting involved, but for anything beyond the .winmd you probably want to run `MIDL` to handle the rest. 🤷‍♂️